### DISCUSSION TITLE
Add meteo

### DISCUSSION THREAD

**Discussion by MXC48-zz** [organicmaps/organicmaps#2223] [Ideas]
In other maps like magic earth I have taken the habit of looking at the integrated weather for the location. I think it would be a good idea to add this to organic map too

**Comment by SirTypesALot** [comment]
This feature would require a somewhat (or periodic) internet connection to get the most up to date weather information. The best way I believe to combat intermittent internet connection would be to make the app to cache the weather data as far into the future the API would allow. The app then would be able to provide some weather data (though it might not be the most up to date), and once the phone gains an internet connection again, it would repeat the process again.

**Reply by biodranik** [reply]
1. Weather changes frequently, so cached info won't have proper accuracy.
2. There are already plenty of other Weather-specialized applications, it would be hard to compete with them.
3. We are focusing on offline experience at the moment.
4. It could be done in the future, when we have enough resources/developers/money, and when we also have other online services: traffic, bookmarks sync/backup, etc.

**Reply by palzkillm** [reply]
> And I think we would need to find a provider that has data licensed in public domain or another free licence.

I think METAR is free..?

**Reply by simonpickering** [reply]
fwiw the UK Met Office provide free data (via a free registration API key): https://www.metoffice.gov.uk/services/data/datapoint/inspire-layers-detailed-documentation

**Comment by trymeouteh** [comment]
Please add some common APIs intregrations for OpenWeatherMap and OpenMeteo for weather radar.
